### ISSUE TITLE
NVDA+8 command missing in "Commands Quick Reference" documentation

### ISSUE THREAD

**Issue by tiago-casal** [nvaccess/nvda#13324]
### Is your feature request related to a problem? Please describe.
The nvda\+8 command appeared in version 2019.2beta1, since then \(currently in version 2021.3.1 at the time this case is posted\) such a command is missing in the \"NVDA Commands Quick Reference\" documentation.
### Describe the solution you'd like
In the user\_docs\/en\/userGuide.t2t file \(and in other languages\), above the line \"\=\=\=\= Automatically set system focus to focusable elements \=\=\=\=\[BrowseModeSettingsAutoFocusFocusableElements\]\", a line with the following content should be included:
\%kc\:setting
